docs/source/daml/intro/5_Restrictions.rst

```diff
@@ -0,0 +1,103 @@
+
+.. literalinclude:: daml/Intro_5_Restrictions.daml
+  :language: daml
+  :start-after: -- TRANSFER_CHOICE_BEGIN
+  :end-before: -- TRANSFER_CHOICE_END
+
+.. literalinclude:: daml/Intro_5_Restrictions.daml
+  :language: daml
+  :start-after: -- TRANSFER_TEST_BEGIN
+  :end-before: -- TRANSFER_TEST_END
+
+In a similar vein, one can write a ``Redeem`` choice, which allows the ``owner`` to redeem an ``Iou`` during business hours on week days. The choice doesn't do anything other than archiving the ``SimpleIou``. The assumption here is that actual cash changes hands off-ledger.
+
+.. literalinclude:: daml/Intro_5_Restrictions.daml
+  :language: daml
+  :start-after: -- REDEEM_CHOICE_BEGIN
+  :end-before: -- REDEEM_CHOICE_END
+
+.. literalinclude:: daml/Intro_5_Restrictions.daml
+  :language: daml
+  :start-after: -- REDEEM_TEST_BEGIN
+  :end-before: -- REDEEM_TEST_END
+
+There are quite a few new time-related functions here so it's time to talk about time on DAML ledgers. There's also quite a lot going on inside the ``do`` block of the ``Redeem`` choice, with several uses of the ``<-`` operator, which deserves a proper explanation at this point.
+
+Time on DAML ledgers
+--------------------
+
+Each transaction on a DAML ledger has two timestamps called the *ledger effective time (LET)* and the *record time (RT)*. The ledger effective time is set by the submitter of a transaction, the record time is set by the consensus protocoll. Each DAML ledger has a policy on the allowed difference between LET and RT called the *skew*. The submitter has to take a good guess at what the record time will be. If it's too far off, the transaction will be rejected.
+
+``getTime`` is an action that gets the LET from the ledger. In the above example, that time is taken apart into day of week and hour of day using standard library functions from ``DA.Date`` and ``DA.Time``. The hour of the day is checked to be in the range from 8 to 18. Suppose now that the ledger had a skew of 10 seconds, but a submission took less than 4 seconds to commit. At 18:00:05, Alice could submit a transaction to redeem an Iou with a LET of 17:59:59. It would be a valid trasnaction and be committed successfully as ``hrs == 17`` and ``LET - RT < 10 seconds``. Time therefore has to be considered slightly fuzzy in DAML, with the fuzzyness depending on the skew parameter.
+
+In scenarios, record and ledger effective time are always equal and they can be set using the functions ``passToDate`` and ``pass``. ``passToDate`` takes a date sets the time to midnight (UTC) of that date. ``pass`` takes a ``Reltime``, a relative time and moves the ledger by that much.
```

Comment:
You might think I'm overusing lists here, but this is a long section that isn't broken up by headings, so it benefits from as much visual interest as we can give it.
```suggestion
Time in scenarios
~~~~~~~~~~~~~~~~~

In scenarios, record and ledger effective time are always equal. You can set them using the following functions: 

- ``passToDate``, which takes a date sets the time to midnight (UTC) of that date and ``pass``
- ``pass``, which takes a ``Reltime`` (a relative time) and moves the ledger by that much
```